### PR TITLE
Cleanup: Minor code cleanup and refactoring

### PR DESCRIPTION
**Pull request by hoh**

### CODE REVIEW

File: src/aleph/vm/pool.py [hunk 1]
```diff
@@ -202,31 +200,43 @@ def forget_vm(self, vm_hash: ItemHash) -> None:
...
+            vm_hash = ItemHash(saved_execution.vm_hash)
+
+            if vm_hash in self.executions:
+                # The execution is already loaded, skip it
                 continue
 
             vm_id = saved_execution.vm_id
             message_dict = json.loads(saved_execution.message)
-            original_dict = json.loads(saved_execution.original_message)
+
```
Comment (nesitor) [inline src/aleph/vm/pool.py:224]: Why remove this one? It is supose to be needed, no?

File: src/aleph/vm/pool.py [hunk 2]
```diff
@@ -202,31 +200,43 @@ def forget_vm(self, vm_hash: ItemHash) -> None:
...
             execution = VmExecution(
-                vm_hash=saved_execution.vm_hash,
+                vm_hash=vm_hash,
                 message=get_message_executable_content(message_dict),
                 original=get_message_executable_content(message_dict),
```
Comment (nesitor) [inline src/aleph/vm/pool.py:228]:
```suggestion
                original=get_message_executable_content(original_dict),
```

File: src/aleph/vm/pool.py (hunk 1 above)
Comment (hoh) [reply src/aleph/vm/pool.py:224]: Unused variable warning in my IDE

File: src/aleph/vm/pool.py (hunk 2 above)
Comment (hoh) [reply src/aleph/vm/pool.py:228]: OK, that is the source of the bug and why I had an unused variable.

**Review by nesitor — APPROVED** [review]

**Pull request merged**